apps/remix-ide/src/app/tabs/runTab/model/dropdownlogic.js

```diff
@@ -50,9 +51,11 @@ class DropdownLogic {
         } catch (e) {
           return cb('Failed to parse the current file as JSON ABI.')
         }
+        _paq.push(['trackEvent', 'udapp', 'AtAddressLoadWithABI'])
         cb(null, 'abi', abi)
       })
     } else {
+      _paq.push(['trackEvent', 'udapp', 'AtAddressLoadWithInstance'])
```

Comment:
ah ok, so this `AtAddressLoadWithInstance` should be `AtAddressLoadWithCompiled` I think.